### ISSUE TITLE
Help with deepgram transcription

### ISSUE THREAD

**Issue by slax81** [livekit/agents#390]
Hello, I've used STT deepgram example from here https://github.com/livekit/agents/tree/main/examples/speech-to-text on a self-hosted LiveKit server and I have a few questions if you can please help me with.
Using agent playground, mic input works and the speech is transcribed, but it is printed in the console output, how can I send it to chat element or some other element on a client app? Can you point me in the right direction? Is it done using client-sdk or should be in the agent code?
I suppose this is the part for transcribed text:

```
    """Forward the transcription to the client and log the transcript in the console"""
    async for ev in stt_stream:
        stt_forwarder.update(ev)
        if ev.type == stt.SpeechEventType.INTERIM_TRANSCRIPT:
            print(ev.alternatives[0].text, end="")
        elif ev.type == stt.SpeechEventType.FINAL_TRANSCRIPT:
            print("\n")
            print("-> ", ev.alternatives[0].text)
```

Basically, I would like to transcribe speech from all participants and send it to one or more specific participants on the client app. We have node js app for video calls.
Sorry if this is not the right place for looking help, but I can't find these answers and understand them from the documentation easily.
If there is some more suitable place where I can post this question, please let me know.
Thank you

**Comment by davidzhao** [comment]
please join the slack community to ask questions.
